src/main/java/com/coherentsolutions/training/aqa/java/web/nikonova/browsers/BrowsersSettings.java

```diff
@@ -0,0 +1,107 @@
+package com.coherentsolutions.training.aqa.java.web.nikonova.browsers;
+
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.io.IOException;
+import java.net.URL;
+
+
+public class BrowsersSettings {
```

Comment:
As Sergey mentioned, would be great to work with ThreadLocal<Webdriver>, this will be thread safe if we are going to run the tests in parallel